assemble/bin/accumulo

```diff
@@ -80,9 +65,6 @@ function main() {
     JAVA=($ACCUMULO_JAVA_PREFIX $JAVA)
```

Comment:
I am ok with splitting on whitespace.  i tested this current code with numactl commands and it worked fine.